wrapper/src/api/tools/cli.ts

```diff
@@ -1,22 +1,63 @@
 import * as dashdash from 'dashdash';
+import path from 'path';
 
-interface IArgs {
-  path?: string;
+interface IArgs extends dashdash.Results {
```

Comment:
In every other file of the api, we do not use "I" as a prefix for interfaces AFAIK. Why is this needed here?

Reply:
done